tensorflow_addons/layers/max_unpooling_2d_v2.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""MaxUnpooling2DV2 operation."""
+
+import tensorflow as tf
+
+from typeguard import typechecked
+from typing import Iterable
+
+from tensorflow_addons.utils.keras_utils import normalize_tuple
+
+
+def _max_unpooling_2d_v2(updates, mask, output_size):
+    """Unpool the outputs of a maximum pooling operation."""
+    output_size_attr = " ".join(["i: %d" % v for v in output_size])
+    experimental_implements = [
+        'name: "addons:MaxUnpooling2DV2"',
```

Comment:
Is it available?

Reply:
Thank you for catching this @fsx950223.
It is not available, and I have removed it from the PR.